src/components/elements/ErrorMessage/ErrorMessage.jsx

```diff
@@ -5,10 +5,10 @@ import { Message } from 'semantic-ui-react';
 
 /**
  * ErrorMessage is a component which uses [Message](https://react.semantic-ui.com/elements/message) component from Semantic-UI-React
- * to display error message.
+ * to display error message. All props supperted by the `Message` component are supported and passed down to it.
```

Comment:
Did you mean
```suggestion
 * to display error message. All props supported by the `Message` component passed down to it.
```
?
I'm not sure if I fixed that properly.

Reply:
Done